cypress-tests/cypress/commands/apiCommands.js

```diff
@@ -125,13 +125,15 @@ Cypress.Commands.add(
   (slug = '',
     workspaceId = Cypress.env("workspaceId"),
     appId = Cypress.env("appId"),
-    componentSelector = "[data-cy='empty-editor-text']"
+    componentSelector = "[data-cy='widget-list-box-form']"
   ) => {
     cy.intercept('GET', '/api/v2/apps/*').as('getAppData');
     cy.window({ log: false }).then((win) => {
       win.localStorage.setItem("walkthroughCompleted", "true");
     });
-    cy.visit(`/${workspaceId}/apps/${appId}${slug}`);
+    cy.visit(`/${workspaceId}/apps/${appId}`);
```

Comment:
I think @ajith-k-v  added the slug on purpose please keep the slug as it is, you can pass it as an empty string and won't cause any issues.